src/components/POLine/POLineAgreementLines/POLineAgreementLinesContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Badge,
+  Loading,
+} from '@folio/stripes/components';
+import {
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+import {
+  AGREEMENT_LINES,
+} from '../../Utils/resources';
+
+import { ACCORDION_ID } from '../const';
+import POLineAgreementLines from './POLineAgreementLines';
+
+const POLineAgreementLinesContainer = ({ lineId, label, mutator }) => {
+  const [agreementLines, setAgreementLines] = useState();
+  const showCallout = useShowCallout();
+
+  useEffect(() => {
+    setAgreementLines();
+
+    mutator.agreementLines.GET()
+      .then(setAgreementLines)
+      .catch(() => {
+        setAgreementLines([]);
+
+        showCallout({ messageId: 'ui-orders.relatedAgreementLines.actions.load.error', type: 'error' });
+      });
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [lineId, showCallout]);
```

Comment:
have you checked dev console and network tab? Also, what is the difference between useEffect and useCallback? And why we don't put showCallout as a dep in useEffect, but put it in useCallback?

Reply:
thanks, removed showCallout. useEffect is for side effects and useCallback - for optimization

Reply:
still, it's pretty important thing, to understand why it works that way,

>  why we don't put showCallout as a dep in useEffect, but put it in useCallback?